e2e/Microsoft.Azure.Devices.E2ETests.NetStandard/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "Microsoft.Azure.Devices.E2ETests.NetStandard": {
+      "commandName": "Project"
+    }
+  }
+}
```

Comment:
new line at eof

Reply:
This is created by VS and not maintained by us, but needed for the project.